backend/terminator/urls.py

```diff
@@ -18,6 +18,7 @@
 from accounts import views as accounts_views
 
 urlpatterns = [
+    path('wordlist/', include('wordlist.urls')),
```

Comment:
I think our design document specifies this URL to be 'api/wordlist/' not 'wordlist/'.

Reply:
Requested change met! 😄